config/webhook/kustomization.yaml

```diff
@@ -1,6 +1,42 @@
+namespace: capi-webhook-system
+
 resources:
 - manifests.yaml
 - service.yaml
+- ../certmanager
+- ../manager
```

Comment:
The manager directory contains the secret with AWS credentials. This and associated logic needs to be moved to `config/default`, such that the deployment in `capi-webhook-system` doesn't get AWS credentials.

Reply:
Resolved!

Reply:
Just checked. Two secrets present in both namespaces.

Reply:
@randomvariable let me know if I did the right thing here: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1575/commits/1219e68e0d4f1657010397a6aa1760330d7c5745